src/schema/home/home_page_artist_module.js

```diff
@@ -77,13 +65,8 @@ export const HomePageArtistModuleType = new GraphQLObjectType({
     },
     results: {
       type: new GraphQLList(Artist.type),
-      resolve: (
-        { key },
-        options,
-        request,
-        { rootValue: { accessToken, userID } }
-      ) => {
-        return HomePageArtistModuleTypes[key].resolve(accessToken, userID)
+      resolve: ({ key }, options, request, { rootValue }) => {
+        return HomePageArtistModuleTypes[key].resolve({ rootValue })
```

Comment:
Here's an example of updating a place that calls some helper methods, by passing in `rootValue`, vs the explicit token/id.